src/EnergyPlus/ChillerElectricASHRAE205.cc

```diff
@@ -54,7 +54,6 @@
 #include <ObjexxFCL/Fmath.hh>
 
 // EnergyPlus Headers
-#include "rs0001.h"
 #include "rs0001_factory.h"
```

Comment:
It would have been nice if these header files were encapsulated in another directory layer.  Seeing a random `#include "rs0001_factory.h"` without context is odd.